TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/linear/LinearInTake.kt

```diff
@@ -0,0 +1,19 @@
+package org.firstinspires.ftc.teamcode.api.linear
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode
+import org.firstinspires.ftc.teamcode.api.InTake
+
+object LinearInTake : LinearAPI() {
```

Comment:
Is there a reason you chose to create two APIs for the intake when the linear slide does not use any LinearOpMode specific functionality? I think they can be merged.